lib/mysqlService.js

```diff
@@ -226,6 +235,21 @@ class MysqlService {
         return Promise.each(this._allPools, pool => pool.endAsync());
     }
 
+    resetPools(delayMs) {
```

Comment:
Add comment explaining why you might want to use this.  Also explain purpose of delayMs.